charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -324,6 +324,10 @@ controller:
   otelTracing: {}
   #  otelServiceName: ebs-csi-controller
   #  otelExporterEndpoint: "http://localhost:4317"
+  dnsConfig:
+   options: 
+     - name : ndots
+       value : "5"
```

Comment:
Hi, thanks for the contribution! Please remove this hardcoded value in the `values.yaml`, we want the default to be left blank.